src/include/sof/ipc.h

```diff
@@ -202,14 +202,25 @@ int ipc_dma_trace_send_position(void);
 /* get posn offset by pipeline. */
 int ipc_get_posn_offset(struct ipc *ipc, struct pipeline *pipe);
 
+enum sof_pm_state {
+	SOF_PM_D0 = 0,	/* Transition to D0, also initial state */
+	SOF_PM_D3,	/* Transition to D3 is requested */
+	SOF_PM_D0ix,	/* Transitions across D0ix substates , incl. D0 are allowed */
+};
+
+enum sof_pm_streaming {
+	SOF_PM_NO_STREAMING = 0,	/* no streaming to the host during D0->D0ix */
+	SOF_PM_STREAMING,	/* streaming to/from host during D0->D0ix is allowed */
+};
+
 /* private data for IPC */
 struct ipc_data {
-	/* DMA */
-	struct dma *dmac;
-	uint8_t *page_table;
-
```

Comment:
@lbetlej what this interntional? it doesnt seem related to this PR?

Reply:
I've reverted the change, this can be cleaned up in another PR